CMakeLists.txt

```diff
@@ -173,6 +173,8 @@ string(REGEX MATCH "Clang" CMAKE_COMPILER_IS_CLANG "${CMAKE_C_COMPILER_ID}")
 
 include(CheckCCompilerFlag)
 
+set(CMAKE_C_STANDARD 99)
```

Comment:
Since this is a backport, I believe [this minimum version](https://github.com/ARMmbed/mbedtls/pull/3641#issuecomment-887502123) no longer applies, so the original check is needed.